package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@google-cloud/tasks",
+  "description": "Cloud Tasks API client for Node.js",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "repository": "googleapis/nodejs-tasks",
+  "main": "src/index.js",
+  "files": [
+    "protos",
```

Comment:
@alexander-fenster why do we ship protos?  Are we doing code generation, or generating stubs at runtime?

Reply:
We generate stubs at runtime. There is no code generation for Javascript.

Reply:
Happens here: https://github.com/googleapis/gax-nodejs/blob/4c8b24272ad11095f5a50c710cef20ca3a868063/src/grpc.ts#L197